crypto/bio/bf_nbio.c

```diff
@@ -32,18 +32,20 @@ typedef struct nbio_test_st {
 } NBIO_TEST;
 
 static const BIO_METHOD methods_nbiof = {
-    BIO_TYPE_NBIO_TEST,
-    "non-blocking IO test filter",
-    bwrite_conv,
-    nbiof_write,
-    bread_conv,
-    nbiof_read,
-    nbiof_puts,
-    nbiof_gets,
-    nbiof_ctrl,
-    nbiof_new,
-    nbiof_free,
-    nbiof_callback_ctrl,
+    .type = BIO_TYPE_NBIO_TEST,
+    .name = "non-blocking IO test filter",
+    .bwrite = bwrite_conv,
+    .bwrite_old = nbiof_write,
+    .bread = bread_conv,
+    .bread_old = nbiof_read,
+    .bputs = nbiof_puts,
+    .bgets = nbiof_gets,
+    .ctrl = nbiof_ctrl,
+    .create = nbiof_new,
+    .destroy = nbiof_free,
+    .callback_ctrl = nbiof_callback_ctrl,
+    .bsendmmsg = NULL,
+    .brecvmmsg = NULL,
 };
 
 const BIO_METHOD *BIO_f_nbio_test(void)
```

Comment:
Was this ever used outside of the s_server/s_client?

Reply:
No, that is the only use. It could be moved out of `crypto/bio` but then we'd have to do the kind of construction done at https://github.com/quictls/quictls/blob/main/apps/s_server.c#L255-L270

I am in favor of removing the `-test_nbio` option and this BIO completely.

Reply:
That's where I was getting at. Let's do this in a separate PR.